autopromote/autopromote.py

```diff
@@ -211,6 +211,20 @@ def get_next_catalog(latest_catalog):
     return None
 
 
+def get_channel_multiplier(plist):
+    """Retrieve the float multiplier for plist's channel. Returns multiplier or 1"""
+
+    channel = plist.get("_metadata", {}).get("channel")
+    if channel is None:
+        return 1
```

Comment:
I think this line (and L223, perhaps) is what @erikng was referring to with [his comment on this PR](https://github.com/Gusto/it-cpe-opensource/pull/26#issuecomment-652065962).

Should this be `return 1.0` for consistent types?